assets/simp-packer-basics.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Picture is very helpful.  Should it mention the tmp dir the user (almost always) needs to supply,  or is that down in the weeds?

Reply:
@lnemsick-simp Thanks!  Yes, I think the TMP_DIR issue isn't so much a big-picture concept as an implementation oddity, and it's documented in other more detail-focused sections of the README.